packages/core/src/texture/RenderTarget.ts

```diff
@@ -153,320 +149,74 @@ export class RenderTarget extends EngineObject {
   }
 
   /**
+   *
    * Get the render color texture by index.
    * @param index
    */
-  public getColorTexture(index: number = 0): RenderColorTexture | null {
+  getColorTexture(index: number = 0): RenderColorTexture | null {
     return this._colorTextures[index];
   }
 
-  /**
-   * Destroy render target.
-   */
-  public destroy(): void {
-    const gl: WebGLRenderingContext & WebGL2RenderingContext = this._rhi.gl;
-
-    gl.deleteFramebuffer(this._frameBuffer);
-    this._depthRenderBuffer && gl.deleteRenderbuffer(this._depthRenderBuffer);
-    this._MSAAFrameBuffer && gl.deleteFramebuffer(this._MSAAFrameBuffer);
-    this._MSAADepthRenderBuffer && gl.deleteRenderbuffer(this._MSAADepthRenderBuffer);
-
-    for (let i = 0; i < this._colorTextures.length; i++) {
-      this._colorTextures[i].destroy();
-    }
-
-    for (let i = 0; i < this._MSAAColorRenderBuffers.length; i++) {
-      gl.deleteRenderbuffer(this._MSAAColorRenderBuffers[i]);
-    }
-
-    this._depthTexture && this._depthTexture.destroy();
-
-    this._frameBuffer = null;
-    this._colorTextures.length = 0;
-    this._depthTexture = null;
-    this._depthRenderBuffer = null;
-    this._MSAAFrameBuffer = null;
-    this._MSAAColorRenderBuffers.length = 0;
-    this._MSAADepthRenderBuffer = null;
-  }
-
-  /**
-   * Activate this RenderTarget.
-   * @remarks
-   * If MSAA is turned on, MSAA FBO is activated, and then this._blitRenderTarget() is performed to exchange FBO.
-   * If MSAA is not turned on, activate the main FBO.
-   */
-  public _activeRenderTarget(): void {
-    const gl: WebGLRenderingContext & WebGL2RenderingContext = this._rhi.gl;
-
-    if (this._MSAAFrameBuffer) {
-      gl.bindFramebuffer(gl.FRAMEBUFFER, this._MSAAFrameBuffer);
-    } else {
-      gl.bindFramebuffer(gl.FRAMEBUFFER, this._frameBuffer);
-    }
-  }
-
   /**
    * Set which face of the cube texture to render to.
    * @param faceIndex - Cube texture face
    */
-  public _setRenderTargetFace(faceIndex: TextureCubeFace = TextureCubeFace.PositiveX): void {
-    const gl: WebGLRenderingContext & WebGL2RenderingContext = this._rhi.gl;
-    const colorTexture = this._colorTextures[0];
-    const depthTexture = this._depthTexture;
-
-    gl.bindFramebuffer(gl.FRAMEBUFFER, this._frameBuffer);
-
-    // Bind render color texture
-    if (colorTexture?._isCube) {
-      gl.framebufferTexture2D(
-        gl.FRAMEBUFFER,
-        gl.COLOR_ATTACHMENT0,
-        gl.TEXTURE_CUBE_MAP_POSITIVE_X + faceIndex,
-        colorTexture._glTexture,
-        0
-      );
-    }
-
-    // Bind depth texture
-    if (depthTexture?._isCube) {
-      gl.framebufferTexture2D(
-        gl.FRAMEBUFFER,
-        depthTexture._formatDetail.attachment,
-        gl.TEXTURE_CUBE_MAP_POSITIVE_X + faceIndex,
-        depthTexture._glTexture,
-        0
-      );
-    }
-
-    // Revert current activated render target
-    this._activeRenderTarget();
+  setRenderTargetFace(faceIndex: TextureCubeFace): void {
```

Comment:
For internal function you must add internal comments and add "_" for function name.